core/src/main/java/org/apache/brooklyn/core/mgmt/BrooklynTags.java

```diff
@@ -51,9 +54,35 @@ public String getContents() {
             return contents;
         }
     }
+
+    public static class InterfacesTag {
+        @JsonProperty
+        final List<Class<?>> interfaces;
+
+        public InterfacesTag(List<Class<?>> interfaces) {
+            this.interfaces = interfaces;
+        }
+
+        public List<Class<?>> getInterfaces() {
+            return interfaces;
+        }
+
+        @Override
+        public boolean equals(Object obj) {
+            return EqualsBuilder.reflectionEquals(this, obj);
```

Comment:
Well, this is not what I saw on my machine, that's why I used this `EqualsBuilder`